packages/asc-ui/src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { FunctionComponent, HTMLAttributes } from 'react'
+import BreadcrumbsStyle from './BreadcrumbsStyle'
+
+const Breadcrumbs: FunctionComponent<HTMLAttributes<HTMLUListElement>> = ({
+  children,
+  className,
+  ...otherProps
+}) => (
+  <BreadcrumbsStyle className={className} {...otherProps}>
```

Comment:
Is the explicit `className` prop needed?

Reply:
stom die was ik vergeten... ik zet hem erin

Reply:
die kan natuurlijk  weg vanwege otherProps

Reply:
hij is weg

Reply:
ook in het BC item